keanu-project/src/test/java/io/improbable/keanu/vertices/VertexLabelTest.java

```diff
@@ -0,0 +1,99 @@
+package io.improbable.keanu.vertices;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.not;
+
+import org.junit.Test;
+
+public class VertexLabelTest {
+    @Test
+    public void byDefaultALabelHasNoNamespace() {
+        VertexLabel foo = new VertexLabel("foo");
+        assertThat(foo.toString(), equalTo(":foo"));
```

Comment:
Do we always want that special character in place?

Reply:
I have no strong opinion about that.

Reply:
I think from the PoV of a user - they'd expect to see the label as they put it in if they aren't using any namespace functionality?

Reply:
fair enough

Reply:
done